modules/benchmark/R/load_data.R

```diff
@@ -43,20 +43,6 @@ load_data <- function(data.path, format, start_year = NA, end_year = NA, site =
     fcn <- match.fun(fcn1)
   } else if (exists(fcn2)) {
     fcn <- match.fun(fcn2)
-  } else if (!exists(fcn1) & !exists(fcn2) & requireNamespace("BrownDog", quietly = TRUE)) { 
-    #To Do: call to DAP to see if conversion to csv is possible
-    #Brown Dog API call through BDFiddle, requires username and password
-    key   <- BrownDog::get_key("https://bd-api.ncsa.illinois.edu",username,password)
-    token <- BrownDog::get_token("https://bd-api.ncsa.illinois.edu",key)
```

Comment:
I think the last remaining step here is to remove `BrownDog` from the benchmark package's DESCRIPTION file, then run `./scripts/generate_dependencies.R` to update the Docker dependency list.

Reply:
@Sweetdevil144 Reminder that this PR needs one more (hopefully quick) change before it's ready to merge.